docs/source/user/extptfm/theory.rst

```diff
@@ -0,0 +1,514 @@
+       \end{bmatrix}
+       ,\qquad
+       \boldsymbol{f}_x=
+       \begin{bmatrix}
+           \boldsymbol{0}  \\
+         \boldsymbol{f}_{r2}\\
+       \end{bmatrix}
+      \end{aligned}
+
+Isolating :math:`\boldsymbol{f}_{C}` from the first block row of Eq. :eq:`moduleB` and
+using the expression of :math:`\boldsymbol{\ddot{x}}_2` from Eq. :eq:`xddot2` leads to:
+
+.. math::
+
+   \begin{aligned}
+       \boldsymbol{f}_{C}
+                  =& \boldsymbol{f}_{r1} - \boldsymbol{M}_{r11}\boldsymbol{\ddot{x}}_1 - \boldsymbol{C}_{r11}\boldsymbol{\dot{x}}_1 - \boldsymbol{C}_{r12}\boldsymbol{\dot{x}}_2 - \boldsymbol{K}_{r11}\boldsymbol{x}_1 \nonumber\\
+                   &- \boldsymbol{M}_{r12}
+   (\boldsymbol{f}_{r2}-\boldsymbol{M}_{r12}^t\boldsymbol{\ddot{x}}_1 -\boldsymbol{C}_{r12}^t\boldsymbol{\dot{x}}_1 -\boldsymbol{C}_{r22}\boldsymbol{\dot{x}}_2-\boldsymbol{K}_{r22} \boldsymbol{x}_2)\end{aligned}
+
+The matrices of for the output :math:`\boldsymbol{y}` are then
+identified as (:cite:`ep-Branlard:2020superelement`):
+
+.. math::
+
+   \begin{aligned}
+       \boldsymbol{C}&=
+       \begin{bmatrix}
+           \boldsymbol{M}_{r12}\boldsymbol{K}_{r22} & \boldsymbol{M}_{r12}\boldsymbol{C}_{r22}-\boldsymbol{C}_{r12}\\
+       \end{bmatrix}
+       ,\qquad
+       \qquad
+       \boldsymbol{f}_y=
+       \begin{bmatrix}
+           \boldsymbol{f}_{r1} - \boldsymbol{M}_{r12}\boldsymbol{f}_{r2}\\
+       \end{bmatrix} \\
+       \boldsymbol{D}&=
+       \begin{bmatrix}
+           -\boldsymbol{K}_{r11} & -\boldsymbol{C}_{r11} + \boldsymbol{M}_{r12}\boldsymbol{C}_{r12}^t & -\boldsymbol{M}_{r11}+\boldsymbol{M}_{r12}\boldsymbol{M}_{r12}^t \\
+       \end{bmatrix}
+      \end{aligned}
+
+All the block matrices and vectors labeled with “r” are provided to the
+module via an input file. At a given time step, the loads,
+:math:`\boldsymbol{f}_r(t)`, are computed by linear interpolation of the
+loads given in the input file, and the state equation, , is solved for
+:math:`\boldsymbol{x}` with the outputs returned to the glue code of
+*OpenFAST*.
+
+The glue code can also perform the linearization of the full system at a
+given time or operating point, using the Jacobians of the state
+equations of each module. Since the formulation of *ExtPtfm* is linear,
+the Jacobian of the state and output equations, with respect to the
+states and inputs of the module, are:
+
+.. math::
+
+   \begin{aligned}
+       \frac{\partial \boldsymbol{X}}{\partial \boldsymbol{x}} = \boldsymbol{A}
+       ,\quad
+       \frac{\partial \boldsymbol{Y}}{\partial \boldsymbol{x}} = \boldsymbol{C}
+       ,\quad
+       \frac{\partial \boldsymbol{X}}{\partial \boldsymbol{u}} = \boldsymbol{B}
+       ,\quad
+       \frac{\partial \boldsymbol{Y}}{\partial \boldsymbol{u}} = \boldsymbol{D}
+      \end{aligned}
+
+The linearization of ExtPtfm was implemented in the module, but some work remains to be 
+done at the glue-code (OpenFAST) level to allow for full system linearization.
```

Comment:
Yes, my understanding from @andrew-platt is that this implementation is not yet complete.  But it would probably not be much work to finalize in the future.

Reply:
Yes I believe the work should be minimal, I had prepared the glue code as well when SubDyn linearization was being implemented.